databricks/koalas/plot.py

```diff
@@ -1684,20 +1692,12 @@ def hist(self, bins=10, **kwds):
         Basic plot.
 
         For Series:
+
         .. plot::
             :context: close-figs
 
             >>> s = ks.Series([1, 3, 2])
             >>> ax = s.plot.hist()
-
-        For DataFrame:
-        .. plot::
-            :context: close-figs
-
-            >>> df = ks.DataFrame({'mass': [0.330, 4.87, 5.97],
-            ...                    'radius': [2439.7, 6051.8, 6378.1]},
-            ...                   index=['Mercury', 'Venus', 'Earth'])
-            >>> plot = df.plot.hist(figsize=(6, 3))
```

Comment:
Why removed the example?

Reply:
Btw, where is this example from? Seems like the original example for hist was:

```py
            >>> df = pd.DataFrame(	
            ...     np.random.randint(1, 7, 6000),	
            ...     columns=['one'])	
            >>> df['two'] = df['one'] + np.random.randint(1, 7, 6000)	
            >>> df = ks.from_pandas(df)	
            >>> ax = df.plot.hist(bins=12, alpha=0.5)	
```


https://github.com/databricks/koalas/pull/1662/files#diff-68e0bb2c8c2742f7ee4d267dbf3d1c68L2143-L2148